packages/docs/docs/fhir-datastore/deleting-data.md

```diff
@@ -27,6 +27,12 @@ However, the original content of the resource is not destroyed. It can still be
 - Reading resource history: `GET Patient/123/_history`
 - Reading a resource version: `GET Patient/123/_history/1`
 
+:::caution Referential Integrity on Deletes
+
+Referential integrity is **not** supported for deletes at this time as it would require checking every resource in a project for inbound references to the deleted resource. Unfortunately, performance tradeoffs forbid Medplum from implementing this.
```

Comment:
```suggestion
Referential integrity is **not** supported for deletes at this time.
```

Once upon a time, we did not support it for performance reasons.  Our database architecture has changed a lot since we first added basic support for referential integrity, and it would now be more feasible for us to implement.  It is still fair to call out that it is unsupported, but I don't think we should point to performance issues .